hs/Response/Draw.hs

```diff
@@ -47,3 +46,12 @@ modePanel = H.div ! A.id "side-panel-wrap" $ do
     H.div ! A.id "finish-region" ! A.class_ "button" $ "finish (f)"
     H.div ! A.id "change-mode" ! A.class_ "mode" $ "SELECT/MOVE (m)"
     H.div ! A.id "erase-mode" ! A.class_ "mode" $ "ERASE (e)"
+    H.input ! A.id "area-of-study"
+            ! A.class_ "course-code"
+            ! A.name "course-code"
+            ! A.placeholder "Enter area of study."
+            ! A.autocomplete "off"
+            ! A.type_ "text"
+            ! A.size "30"
+    H.div ! A.id "submit-gid" ! A.class_ "button" $ "Submit"
```

Comment:
I think you need to leave them... @Ian-Stewart-Binks or @R0ss, either of you could actually try to change it yourself.